ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityDateAnswer.component.ts

```diff
@@ -33,7 +33,11 @@ export class ActivityDateAnswer {
     constructor(private logger: LoggingService) { }
 
     public handleChange(value: DatePickerValue | null): void {
-        let dateValue: DatePickerValue | null = null;
+        let dateValue: DatePickerValue | null = {
```

Comment:
`let dateValue: DatePickerValue = {...};`
can't be a null type anymore

Reply:
Correct! I'll fix that.